sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 # Release History
 
 ## 4.4.0-beta.1 (2020-08-27)
+### New Features
 * Added new API to efficiently load many documents (via list of pk/id pairs or all documents for a set ok pk values)
 * Enabled query metrics by default
-* Fixed NPE in GatewayAddressCache
+### Key Bug Fixes
+* Fixed NPE in `GatewayAddressCache`
+* Fixing query metric issue for zero item response
```

Comment:
Improved performance (reduced CPU usage) for address parsing and Master-Key authentication